src/main/java/unicash/ui/TransactionCard.java

```diff
@@ -159,8 +179,45 @@ private void categoriesStyleFormatter() {
                 .substring(1, categoriesToStringLength - 1);
 
         String categoriesToStringWithHashTag = "#" + trimmedCategoriesToString;
-        categories.setText(categoriesToStringWithHashTag);
-        categories.setStyle(FONT_STYLE_BOLD);
+        firstCategory.setText(categoriesToStringWithHashTag);
+        firstCategory.setStyle(FONT_STYLE_BOLD);
+        firstCategory.setText("testing");
+        firstCategory.setStyle("-fx-background-color: #FFD43E");
+    }
+
+    /**
+     * For color coding every category using its hash value to generate a distinct
+     * and unique color without having to store additional color information inside
+     * the {@code Category} class.
+     **/
+    private void discreteCategoriesStyleFormatter() {
+        UniqueCategoryList categoryUniqueList = transaction.getCategories();
+        ObservableList<Category> categoryObservableList =
+                categoryUniqueList.asUnmodifiableObservableList();
+        Iterator<Category> categoryIterator = categoryUniqueList.iterator();
+        ArrayList<Category> categoryArrayList = new ArrayList<>();
+
+        ArrayList<Label> labelArrayList = new ArrayList<>(
+                Arrays.asList(firstCategory, secondCategory, thirdCategory,
+                        fourthCategory, fifthCategory)
+        );
```

Comment:
Could the declaration be 

```java
List<Label> labelArrayList = Arrays.asList(firstCategory, secondCategory, thirdCategory, fourthCategory, fifthCategory)
``` 

instead? Since it doesn't seem like  you're modifying this array list so you can use a `List` instead of `ArrayList`